docs/part-4-labeling-the-data-and-retrain/chapter-20-link-the-model-to-label-studio.md

```diff
@@ -0,0 +1,260 @@
+                    {
+                        "value": {"choices": [prediction]},
+                        "id": str(uuid.uuid4()),
+                        "from_name": "choice",
+                        "to_name": "image",
+                        "type": "choices",
+                    }
+                ],
+            }
+        ],
+    }
+```
+
+This file is structured in 3 parts:
+
+1. Defines some useful constants.
+2. Creates the FastAPI app and loads the model similarly to BentoML
+    `src/serve.py`.
+3. Defines the API endpoints required by Label Studio.
+
+!!! note
+
+    We do nothing with the `/setup` and `/webhook` endpoints as they are not
+    required for the model to work with Label Studio. If you would like to go
+    further, please refer the
+    [Label Studio ML integration documentation](https://labelstud.io/guide/ml).
+
+### Run the Model API
+
+Run the model API by executing the following command:
+
+```sh title="Execute the following command(s) in a terminal"
+fastapi run src/serve_label_studio.py
+```
+
+The FastAPI server will start at <http://localhost:8000> and you can view the
+documentation at <http://localhost:8000/docs>.
+
+Make sure to keep the server running in the background for the next steps.
+
+### Add the Model to Label Studio
+
+Make sure Label Studio is running at <http://localhost:8080>.
+
+1. Click on the `Settings` button on the top right corner of the screen and
+   select `Model` from the sidebar.
+
+    ![Label Studio Settings Model](../assets/images/label-studio-settings-model.png)
+
+2. Click on the `Connect Model` button and enter the following details:
+
+    - **Name**: `celestial_bodies_classifier_model`
+    - **Backend URL**: `http://localhost:8000`
+
+    It should look like this:
+
+    ![Label Studio Connect Model](../assets/images/label-studio-connect-model.png)
+
+3. Click on `Validate and Save` to add the model.
+
+You should now see your model with a status of `Connected`:
+
+![Label Studio Model Connected](../assets/images/label-studio-model-connected.png)
+
+### Label with the Model
+
+Now that the model is connected, you can start labeling the data.
+
+1. **Click on your project name** on the top navigation bar to go back to the
+   project view.
+
+    ![Label Studio Label Project View](../assets/images/label-studio-label-project-view.png)
+
+2. Click on the `Label All Tasks` button to start labeling the data. This will
+   resume the labeling process from where you left off.
+
+    You should now see the image has been labeled automatically by the model:
+
+    ![Label Studio Labeled Image](../assets/images/label-studio-labeled-image.png)
+
+3. Review the prediction and make any necessary corrections.
+
+4. Click on the `Submit` button to save the annotation.
+
+5. Continue labeling the data until you have labeled all the images.
+
+And that's it! You can view the annotations in the project view:
+
+![Label Studio Annotations](../assets/images/label-studio-annotations.png)
+
```

Comment:
This is working great!

As a general remark, it might however be useful to add a note/warning about the danger intrinsic to AI assisted labelling. Human labellers might become lazy and rely too much on the given prediction, and accept it without correcting it - especially in the case of labelling a lot of data on a long time period.

As we can see, some images are wrongly predicted (some og Neptune images are predicted as Uranus). We might introduce the more general concept of cross labelling (having more than one person labelling the same data, then checking if the labels match or not) etc.